server/config/routes.js

```diff
@@ -323,7 +301,7 @@ exports.configure = function configure(app) {
   /* Depot distributions routes */
   app.get('/depots/:depotId/distributions', depots.listDistributions);
   app.get('/depots/:depotId/distributions/:uuid', depots.detailDistributions);
-  app.post('/depots/:depotId/distributions', depots.createDistributions);
+  // app.post('/depots/:depotId/distributions', depots.createDistributions);
```

Comment:
It looks like the `createDistributions` method has been removed from depots, this can probably be fully removed.